fem/hyperbolic.hpp

```diff
@@ -19,17 +19,17 @@ namespace mfem
 {
 
 // This file contains general hyperbolic conservation element/face form
-// integrators. HyperbolicFormIntegrator and RiemannSolver are defined.
+// integrators. HyperbolicFormIntegrator and NumericalFlux are defined.
 //
 // HyperbolicFormIntegrator is a NonlinearFormIntegrator that implements
 // element weak divergence and interface flux
 //
-//     ∫_T F(u):∇v,   -∫_e F̂(u)⋅[[v]]
+//     ∫_K F(u):∇v,   -∫_f F̂(u)⋅n[v]
 //
-// Here, T is an element, e is an edge, and [[⋅]] is jump. This form integrator
-// is coupled with RiemannSolver that implements the numerical flux F̂. For
-// RiemannSolver, the Rusanov flux, also known as local Lax-Friedrichs flux, is
-// provided.
+// Here, K is an element, f is a face, n normal and [⋅] is jump. This form
+// integrator is coupled with NumericalFlux that implements the numerical flux
+// F̂. For NumericalFlux, the Rusanov flux, also known as local Lax-Friedrichs
+// flux, or Godunov flux are provided.
```

Comment:
Godunov flux only provided for scalar conservation laws.

Reply:
Mentioned it is component-wise.